sh/README.md

```diff
@@ -0,0 +1,13 @@
+## Getting Started
+
```

Comment:
Explain how to setup in Windows, Linux and MacOS. Windows has set up a new bash installer. Make sure you mention about it.